src/collective/sidebar/browser/sidebar.py

```diff
@@ -47,6 +47,15 @@ def get_portal_url(self):
         """
         return api.portal.get().absolute_url()
 
+    def get_sidebar_links(self):
```

Comment:
I would like to keep it sidebar_links if possible since we have multiple action categories, and to prevent any confusion the method is named the same as the category it fetches. The sidebar_links. I think static_links is too ambiguous... My 2c!